### ISSUE TITLE
LittleTiles feature backport

### ISSUE THREAD

**Issue by xebecnyc** [GTNewHorizons/GT-New-Horizons-Modpack#18063]
### Your GTNH Discord Username

xebecnyc

### Your Pack Version

nightly 712/ beta 3

### Your Proposal

add some features from 1.12.2 little tiles such as the better UI with with the selection wheel and more bit types because as it is right now littletiles in the pack just sits there and no one uses it because its kinda bad and very inconvenient to use compared to the 1.12.2 version or newer

### Your Goal

Making little detail oriented builds easier and more stuff that requires small details in builds

### Your Vision

changing some features of the current LittleTiles to match the newest one would make alot of builders happy.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this feature please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [X] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [X] I understand this change request may not attract enough attention and thus not be implemented.
- [X] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [X] I believe this feature would make the pack better.

**Comment by Ultraxite** [comment]
I agree with this too, the main feature that would help is the ease of use/ability to quickly expand LT structures, and the LT slopes.

good luck finding someone who a) knows enough about rendering code; b) cares enough about building/aesthetics to want this; c) has time to do this; d) plays GTNH/1.7.10 packs.

**Comment by Flakxo** [comment]
littletiles is genuinely one of the best building mods in my opinion and itd be amazing to see its modern features backported(mainly the perfect slopes), but the dev has dropped all support for 1.7.10, hopefully theyd allow someone else to update it, if anyone is even willing to

**Comment by jkieberking** [comment]
started looking into this - have an initial "working" version of the 1.12 version of the chisel item and using it to create arbitrary slopes:

https://github.com/user-attachments/assets/5f20df69-2ec0-4819-b316-27c096e5a91b

**Comment by DarkShadow44** [comment]
I'm on it, still waiting for my PRs to be reviewed

**Comment by Ngar1** [comment]
@DarkShadow44 how is the startus of your pack port?

**Comment by DarkShadow44** [comment]
@Ngar1 Pretty good. But also a lot to do still. We got

- Chisel
- Grid Alignment
- Placement Modes (Normal, Stencil, Fill, Overwrite)
- Block picking
- Box / Slopes / Round Slopes / Pillars
- Rotations
